COTLMPServer/Messages/Message.cs

```diff
@@ -0,0 +1,123 @@
+﻿/*
+ * PROJECT:     Cult of the Lamb Multiplayer Mod
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     Define the Message class
+ * COPYRIGHT:	Copyright 2025 Neco-Arc <neco-arc@inbox.ru>
+ */
+
+/* IMPORTS ********************************************************************/
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+
+/* CLASSES & CODE *************************************************************/
+
+/**
+ * @brief
+ * Contains all classes/structs associated with network messages
+ */
+namespace COTLMPServer.Messages
+{
+    /**
+     * @brief
+     * Message header
+     * 
+     * @field Type
+     * Message type
+     * 
+     * @field ID
+     * Client ID (< 0 if none)
+     * 
+     * @field Data
+     * Actual message data (depends on the message type)
+     * 
+     * @field MagicNumber
+     * Magic number to identify messages
+     */
+    public sealed class Message
+    {
+        public MessageType Type;
+        public int ID;
+        public byte[] Data;
+        public const int MagicNumber = 0x173495;

```

Comment:
Maybe define a human-readable name for this magic construct? So that others understand what's the purpose of it.